src/components/FormImage/FormImage.vue

```diff
@@ -21,6 +21,11 @@
             v-if="preview"
             class="form-image-btn-edit"
         />
+
+        <slot
+            v-if="!preview"
```

Comment:
Would this do the same?
```suggestion
        <slot
            v-else
```